docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Gift Cards
+---
+
+:::note
+Gift cards are available in Saleor 3.1.
+:::
+
+This page contains information on how to manage gift cards by selecting a specific currency, sending them to customers, or setting expiry dates.
+
+## Introduction
+Gift cards are timeless and universal gifts. They are suitable for customers, employees, CEOs, business partners, or family members. Issuing gift cards can increase your sales and provide a better customer experience.
+
+## How to
+### How to create gift cards?
+Go to the Gift Card page of the Catalog section.
+
+:::note
+If you would like a gift card to act as a payment method, first you need to create a gift card [product type](dashboard/configuration/products.md).
```

Comment:
@karolkielecki this information is not correct. You can use any viable gift card as payment method. You need a product type in order to sell a gift card product in your store.